backend/src/routes/metadata-routes.ts

```
@@ -217,6 +221,103 @@ const metadataRoutes: FastifyPluginAsync = async (server) => {
+      printRequest('/metadata/v3', request, server.log);
+      const query: any = request.query;
+      const chainId = query['chainId'] ?? 137;
+      const api_key = query['apiKey'];
+      if (!api_key || typeof(api_key) !== "string")
+        return reply.code(ReturnCode.FAILURE).send({ error: ErrorMessage.INVALID_API_KEY })
+      if (!chainId)
+        return reply.code(ReturnCode.FAILURE).send({ error: ErrorMessage.INVALID_DATA })
+      let customPaymasters = [];
+      let multiTokenPaymasters = [];
+      let privateKey = '';
+      let sponsorName = '', sponsorImage = '';
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)